CONTRIBUTING.rst

```diff
@@ -6,22 +6,89 @@ are two sorts of contributions:
 
 - Changes to the skeleton structure should be made on a branch, then a
   PR_ raised back to the ``skeleton`` branch.
-- Changes to the docs or commandline tool should be made directly on master
+- Changes to the docs or commandline tool should be made directly on main
 
-.. _PR: https://github.com/dls-controls/dls-python3-skeleton/pulls
+.. _PR:  https://github.com/epics-containers/python3-pip-skeleton/pulls
+.. _GitHub: https://github.com/epics-containers/python3-pip-skeleton/issues
 
 Running the tests
 -----------------
 
-Both the ``skeleton`` and ``master`` branch have different tests, but
-the process to run them is the same::
+Both the ``skeleton`` and ``main`` branch have different tests, but
+the process to run them is the same. You can run in a container using the 
+supplied devcontainer or you can run them locally.
 
-    $ git clone git://github.com/dls-controls/dls-python3-skeleton.git
-    $ cd dls-python3-skeleton
-    $ pipenv install --dev
-    $ pipenv run tests
+To run in a container
+~~~~~~~~~~~~~~~~~~~~~
 
-Please note that Pipfile.lock is only committed on ``master``, not on
+Use vscode devcontainer as follows::
+
+    $ git clone git://github.com/epics-containers/python3-pip-skeleton.git
+    $ vscode python3-pip-skeleton
+    Click on 'Reopen in Container' when prompted
+    In a vscode Terminal:
+    $ tox -p
+
+
+To run locally
+~~~~~~~~~~~~~~
+
+Get the source source code and run the unit tests directly
+on your workstation as follows::
+
+    $ git clone git://github.com/epics-containers/python3-pip-skeleton.git
+    $ cd python3-pip-skeleton
+    $ virtualenv .venv
+    $ source .venv/bin/activate
+    $ pip install -e .[dev]
+    $ tox -p 
+
+In both cases tox -p runs in parallel the following checks:
+
+  - Build Spinx Documentation
+  - run pytest on all tests in ./tests
+  - run mypy linting on all files in ./src ./tests
+  - run pre-commit checks:
+
+    - run flake8 style checks against all source
+    - run black formatting checks against all source
+
+Use vscode devcontainer as follows::
+
+    $ git clone git://github.com/epics-containers/python3-pip-skeleton.git
+    $ vscode python3-pip-skeleton
+    Click on 'Reopen in Container' when prompted
+    In a vscode Terminal:
+    $ tox -p
+
+
+To run locally
+~~~~~~~~~~~~~~
+
+Get the source source code and run the unit tests directly
+on your workstation as follows::
+
+    $ git clone git://github.com/epics-containers/python3-pip-skeleton.git
+    $ cd python3-pip-skeleton
+    $ virtualenv .venv
+    $ s
```

Comment:
typo?

Reply:
Sorry Andy, this was to the wrong fork. I was working on the pip only workflow version which we have given a new name https://github.com/epics-containers/python3-pip-skeleton